### ISSUE TITLE
As a developer, when I visit a tribe page, I want to see the resources that I need for building my project.

### ISSUE THREAD

**Issue by willsputra** [gitcoinco/web#7570]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a developer, when I visit a tribe page, I want to see the resources that I need for building my project.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: When I visit a tribe page, it's not clear to me what tribes is about, what I can do in it, and how it can help me in building cool projects.

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Description
Let's say I'm a developer and I want to build cool projects. I've heard that Matic is cool, and I want to try building a project that uses Matic. So I go to Matic tribes page (https://gitcoin.co/maticnetwork/) to find out more about them. What do I see?

### Current Behavior
[comment]: # (Describe what actually happened.)
When I go to Matic tribes page, I immediately see the Matic townsquare with zero context. Where do I start? How do I find resources to start building a project with Matic? I see a lot of "Worker applied: " on Matic townsquare, what are these about? There's no about page, and the sidebar doesn't have any explanation/description about Matic.

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
When I go to Matic tribes page, I'd love to see: (a) further explanation on what Matic is and why Matic is cool and (b) resources on Matic that's relevant on each step of my project development journey.

(a) How might we show further explanation on what Matic is and why Matic is cool?
• Orgs description
• How much they've funded people on Gitcoin
• What cool projects are being built with Matic (show which ones are successful grants, and which ones are hackathon winners) https://github.com/gitcoinco/web/issues/7550

(b) How might we show resources on Matic that's relevant on each step of a dev's project development journey?
• Getting started (Show quests about Matic, getting started docs about Matic...)
• Examples in projects (Show some examples of projects that have implemented Matic, and how they go about implementing it)
• Ask the community (Tribes townsquare. Circle?)
• If you want to showcase your project and win stuff, participate in a Hackathon
• If you want to get funding on your project, convert to Grant
• etc.

Basically **how do we turn a tribe page into a one-stop destination for devs who are building cool projects using the tribe's tech?**

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)
- [ ] Brainstorm how the tribes page can help surface resources that devs need for building their projects.
- [ ] Design

**Comment by willsputra** [comment]
@thesachinmittal @sanchaymittal I created this ticket based on our discussion on tribes the other day. let's continue brainstorming here! 🙂

**Comment by aamustapha** [comment]
@octavioamu @willsputra I think a UI Design would be needed for this.

**Comment by thesachinmittal** [comment]
This is awesome @willsputra , i have created an umbrella ticket on, [types of interaction at Gitcoin.](https://github.com/gitcoinco/web/issues/7709)
